### DISCUSSION TITLE
installation_method variable

### DISCUSSION THREAD

**Discussion by redy01** [autobase-tech/autobase#499] [General]
Just wonder why installation_method set to repo is mixed between installation from repository (RPM) and git repository?
BTW packages consul, etcd, haproxy, keepalived and vip-manager are available via PostgreSQL common extras repo as RPM packages what can simplify (offline) installation process on EL systems.

**Comment by vitabaks** [comment]
I'm thinking about removing `installation_method` in a future release, since offline installation is not so delicate functionality (since you can use a proxy server), but maintaining this method complicates the code.

**Reply by redy01** [reply]
Most of DB production installations have no direct or proxy access to outside world, so offline installations are mandatory in my opinion. If not from own local rpm,deb repo what should be preferred all packages can be downloaded to installation host and used from there.

**Reply by vitabaks** [reply]
I disagree. I have a lot of experience working with databases (more than 10 years) and almost all my servers have access to a proxy server or Internet access and at the same time there is no access to the database FROM the Internet.
